test/util/utilsSpec.ts

```diff
@@ -576,4 +622,9 @@ describe('parseUrl adds all missing parts of the url', () => {
     expect(utils.parseUrl('localhost?query=value#login', '/deepstream'))
       .to.equal('ws://localhost/deepstream?query=value#login')
   })
+
+  it('rejects urls with @no-host', () => {
```

Comment:
I would stop using @ notations in tests, never seen that standard anwhere before.

Reply:
I agree.

Those are basically meaningless leftovers that I forgot to remove, I was using them with `--grep` when I was focusing on these cases..